Sources/Components/BottomSheet/Transition/BottomSheetPresentationController.swift

```diff
@@ -27,6 +27,13 @@ class BottomSheetPresentationController: UIPresentationController {
         return .overCurrentContext
     }
 
+    private lazy var dimView: UIView = {
+        let view = UIView(frame: UIScreen.main.bounds)
```

Comment:
Can we use auto layout instead? Because I think it's gonna break if you rotate to landscape on iPad, for example.

Reply:
Sure